### PR TITLE
ADA-1194 - Add padding to title and subtitle

### PR DESCRIPTION
**Pull request by valentinStoicaK**
For Zoom 200% + the Title and subtitle of this modal was overlapping with the X Close button. I've used a padding-right of 44px that solved the issue

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@playkit-js/moderation",
-  "version": "3.2.17",
+  "version": "3.2.19",
```
Comment (semarche-kaltura) [inline package.json:3]: version sets by `prod` script in github actions, so to avoid conflicts we shouldn't modify value manually

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### 3.2.19 (2024-06-30)
+
+### 3.2.18 (2024-06-16)
+
```
Comment (semarche-kaltura) [inline CHANGELOG.md:8]: `CHANGELOG.md` generates automatically, so to avoid conflicts we shouldn't modify value manually

File: package.json [hunk 3]
```diff
@@ -80,7 +80,7 @@
   "kaltura": {
     "name": "playkit-js-moderation",
     "dependencies": {
-      "playkit-ui-managers": "1.5.4"
+      "playkit-ui-managers": "1.5.6"
```
Comment (SivanA-Kaltura) [inline package.json:83]: why is this needed ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

**Comment by valentinStoicaK** [comment]
I'm closing this and trying to open a fresh one
